src/storage/BaseProcessor.h

```diff
@@ -80,7 +80,10 @@ class BaseProcessor {
 
     void handleLeaderChanged(GraphSpaceID spaceId, PartitionID partId);
 
-    void handleAsync(GraphSpaceID spaceId, PartitionID partId, kvstore::ResultCode code);
+    void handleAsync(GraphSpaceID spaceId,
+                     PartitionID partId,
+                     kvstore::ResultCode code,
+                     std::shared_ptr<int32_t> requestPtr);
```

Comment:
maybe use the pointer to the on flying request in storage env directly?

Reply:
Because we will keep on flying request number in `requestPtr`  which will be remove in the callback.